src/Microsoft.IdentityModel.Tokens/CryptoProviderFactory.cs

```diff
@@ -84,6 +84,15 @@ static CryptoProviderFactory()
         /// </summary>
         public CryptoProviderFactory()
         {
+            CryptoProviderCache = new InMemoryCryptoProviderCache();
+        }
+
+        /// <summary>
+        /// Default constructor for <see cref="CryptoProviderFactory"/>.
+        /// </summary>
+        public CryptoProviderFactory(CryptoProviderCache cache)
```

Comment:
Should this be public? I think yes, it allows ctor injection as an extensibility point. other thoughts?